quickwit-swim/src/membership.rs

```diff
@@ -119,7 +122,24 @@ impl ArtilleryMemberList {
         (suspect_members, down_members)
     }
 
-    pub fn mark_node_alive(&mut self, src_addr: &SocketAddr) -> Option<ArtilleryMember> {
+    // make sure node id is set correctly for the host (in case it changes)
+    pub fn ensure_node_id(&mut self, src_addr: SocketAddr, node_id: &str) {
```

Comment:
`ensure` suggests there is some kind of check in your method.
`set` would be better.

`set_node_id(_, _)`